sites/upsun/src/development/variables/use-variables.md

```diff
@@ -60,7 +60,7 @@ To access environment variables in your app, use a built-in method for the given
 * PHP: The [`getenv()` function](https://www.php.net/manual/en/function.getenv.php)
 * Python: The [`os.environ` object](https://docs.python.org/3/library/os.html#os.environ)
 * Node.js: The [`process.env` object](https://nodejs.org/api/process.html#process_process_env)
-* Ruby: The [`ENV` accessor](https://ruby-doc.org/current/ENV.html)
+* Ruby: The `ENV` accessor <!-- 20250522 - ruby-doc.org site is not reachable (https://ruby-doc.org/current/ENV.html) -->
```

Comment:
https://docs.ruby-lang.org/en/master/ENV.html ?

Reply:
i thing they change their architecture and miss some redirect.
Maybe you could contact them and sell them some consulting days on rediction.io :P